src/libraries/apiclient/sync/mediasync.js

```diff
@@ -2,9 +2,9 @@ define(["localassetmanager"], function(localassetmanager) {
     "use strict";
 
     function processDownloadStatus(apiClient, serverInfo, options) {
-        return console.log("[mediasync] Begin processDownloadStatus"), localassetmanager.resyncTransfers().then(function() {
+        return console.debug("mediasync: begin processDownloadStatus"), localassetmanager.resyncTransfers().then(function() {
             return localassetmanager.getServerItems(serverInfo.Id).then(function(items) {
-                console.log("[mediasync] Begin processDownloadStatus getServerItems completed");
+                console.debug("mediasync: begin processDownloadStatus getServerItems completed");
```

Comment:
You really didn't need to do any of these, we need to remove the API client from source eventually anyway.